### PR TITLE
Updates on Readme

### LINKED ISSUE #5: Document repository
Add needed documentation

### PR DESCRIPTION
**Pull request by cmoralesdiego**
Updating Readme.md to allows the users to understand how to work with Smart Contracts. It is a WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,36 @@ The previous contracts use some libraries which are:
...
+
+### Remix
+To work with remix, first of all you need to have installed remixd in your system, for this, you have to write the following command:
+`` npm i -g @remix-project/remixd `` doing this you have remixd installed. Once is done you have to write the next command to run the daemon pointing to the directory where the smart contracts are: 
+``
+remixd -s /home/ubuntu/Escritorio/temp-alastriaID-truffle-contracts/contracts/ http://remix.ethereum.org
+``
+Once it is done the previous step, in your browser you have to type remix.ethereum.org and you have to connect the Remixd plugin
+
+![Remixd Plugin](https://github.com/cmoralesdiego/images/blob/main/CaptureRemixdplugin.JPG)
```
Comment (nefera606) [inline README.md:55]: Can we set the images from the alastria repo instead? to maintain within itself

File: README.md [hunk 2]
```diff
@@ -33,3 +33,36 @@ The previous contracts use some libraries which are:
...
+[WARN] You may now only use IDE at http://remix.ethereum.org to connect to that instance
+setup notifications for /home/ubuntu/Escritorio/temp-alastriaID-truffle-contracts/contracts/
+
+````
+As well of you click on the first icon where the smart contracts are, they will appear on this tab.
+![Remix Contracts](https://github.com/cmoralesdiego/images/blob/main/CaptureContracts.JPG)
+
+Then you can work with the Smart Contracts, deploying firstly AlastriaIdentityManager
+
+### Ganache + Truffle
```
Comment (nefera606) [inline README.md]: There is also a docker commando from the package to help on this task

File: README.md [hunk 3]
```diff
@@ -33,3 +33,36 @@ The previous contracts use some libraries which are:
 |Eidas.sol|	It manages Eidas level of assurance for credentials |
 |Owned.sol|	It assures that just the account which deployed a contract can update the version |
 
+## Deploy Contracts
+In this section it will be possible to see how to work with Smart Contracts in different environments which are: Remix, Ganache + Truffle and in a test environment
+
+### Remix
+To work with remix, first of all you need to have installed remixd in your system, for this, you have to write the following command:
+`` npm i -g @remix-project/remixd `` doing this you have remixd installed. Once is done you have to write the next command to run the daemon pointing to the directory where the smart contracts are:
```
Comment (nefera606) [inline README.md:49]: Truffle is also installed in the repo if global dependency is not wanted

File: README.md (hunk 2 above)
Comment (cmoralesdiego) [reply README.md]: This is f** great and wonderful. Thanks @nefera606 , I modify the readme.

File: README.md (hunk 1 above)
Comment (cmoralesdiego) [reply README.md:55]: Where is the best place to upload the images?

Comment (VictorNS69) [reply README.md:55]: Maybe a new directory `/docs/assets/images` ? What do you think @nefera606?

**Comment by cmoralesdiego** [comment]
Updates with scripts from package.json

**Comment by cmoralesdiego** [comment]
Changes on package.json as it is written badly one of the scripts

File: README.md (hunk 1 above)
Comment (nefera606) [reply README.md:55]: I think just /images or /assets. it's only for readme purpose, if we will have more docs, could be, but I think this is not he perfect repo

**Comment by VictorNS69** [comment]
I change the target branch to be able to edit it better in this repository.

**Pull request merged**